plot_continuous_monitor.py

```diff
@@ -3,46 +3,26 @@
 import json
 import os
 import subprocess
-import sys
+import matplotlib.pyplot as plt
```

Comment:
can you use the full name (pyplot) instead of re-naming it? that generally makes it easier to read code and understand where library calls are coming from